docs/source/basics.rst

```diff
@@ -39,6 +39,13 @@ context results in a type check error:
 
    a = p()   # Type check error: p has None return value
 
+Functions with keyword arguments can be annotated as follows:
```

Comment:
I'd change this to "Arguments with default values can be ..." -- that's technically more correct. If you make that change I'll merge this.